fundbot/util.py

```diff
@@ -3,3 +3,11 @@
 
 def get_random_dt():
     return int(time.time()*1000 - random.randint(1, 500))
+
+def get_timestamp_from_dt(date):
+    try:
+        time_array = time.strptime(date, "%Y-%m-%d")
+        time_stamp = int(time.mktime(time_array)*1000 - random.randint(1, 500))
+    except Exception as e:
```

Comment:
meanless `try ... except` ?

Reply:
Exception will be caught in func get_fund_trend_data

Reply:
if so, you don't need to write `try ... except`.